CHANGELOG.md

```diff
@@ -4,6 +4,78 @@ All notable, unreleased changes to this project will be documented in this file.
 
 # 3.17.0 [Unreleased]
 
+### Highlights
+
+- Introduce promotions, that allow applying discounts based on certain conditions. - #12980 by @IKarbowiak @zedzior
```

Comment:
Maybe lets add one more line describing that promotions are replacement for sales and covers its functionalities, like below:
```suggestion
- Introduce promotions, that allow applying discounts based on certain conditions. - #12980 by @IKarbowiak @zedzior
   The promotions are replacement for current sales and covers all sales functionalities and provide additional features.
```

Reply:
good point